packages/browser/src/components/DataBrowser/DataBrowser.js

```diff
@@ -75,6 +76,8 @@ class DataBrowser extends Component<Props> {
 			indexes,
 			headers,
 		} = this.props;
+		let { showClone } = this.props;
```

Comment:
@ShahAnuj2610 can you rename this to `hasCloneApp` to make it more clear.

Reply:
done